tests/timeseries/mock_microgrid.py

```diff
@@ -80,7 +80,9 @@ async def start(self) -> Tuple[Sender[ComponentMetricRequest], ChannelRegistry]:
         """
         await self._server.start()
         await asyncio.sleep(0.1)
-        return await self._init_client_and_actors()
+        ret = await self._init_client_and_actors()
+        await asyncio.sleep(0.1)
+        return ret
```

Comment:
Please try to remove first sleep() (aster server.start()) and check if that works.

Reply:
will clean up as part of #155,  because it is quite slow to test.